account_invoice_send_mail_queued/tests/test_asynchronous_invoice_email.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import odoo.tests.common as common
+
+
+class TestAsynchronousInvoiceEmail(common.TransactionCase):
+    def setUp(self):
+        super(TestAsynchronousInvoiceEmail, self).setUp()
+        self.account_invoice_send_obj = self.env["account.invoice.send"]
+        self.job_obj = self.env["queue.job"]
+        self.invoice_obj = self.env["account.move"]
+
+    def test_delay_send_invoices(self):
+        invoice_1 = self.env.ref("l10n_generic_coa.demo_invoice_1").id
```

Comment:
Don't hardcode this. You are forcing to have `l10n_generic_coa` installed, which is not the case on integration CIs.

Reply:
Done